README.md

```diff
@@ -1,67 +1,129 @@
 
 # Sketch
 
-The work of collecting information about an entity can organized into the following stages
+One of the biggest challenges to developing dossiers is **managing and archiving inferences**. At every level of data collection – from text extraction, to entity resolution, to drawing org charts – there are different kinds of claims that are made about the relationship between sources (data) and entities (the world). Our goal is to develop a consistent language to describe these different kinds of inferences, and use that language to construct a data model that would allow the users of a dossier tool to efficiently collect and audit these claims. 
+
+We believe that the work of collecting information about an entity can be roughly organized into **five levels of inference**. At each level, we want to collect metadata about:
+
+- Who or what made the inference
+- When and where the inference was made
+- What reasoning process produced the inference (be it human- or machine-driven)
+
+### 1. Text: *Collecting source artifacts*
+
+Every claim about the world starts with a source artifact, or *Text*, that contains information that humans can interpret as language. For us, Texts are most often different types of files: word processing documents, PDFs, markup, spreadsheets, etc. If we were building a dossier about security forces, for example, Texts might be audio testimony from witnesses and victims, news articles about security force activity, or scanned government documents.
+
+At the level of *Text*, the processes that can produce inference include:
+
+- Human collection
+- Web scraping
+- FOIA requests
+- Bulk downloads
+
+### 2. Tokens: *Extracting units of meaning from Text*
+
+Assuming that we are working with physical artifacts that we believe to be representations of language, our next step is to interpret these artifacts as a sequence of *Tokens*, the most basic unit of meaning in a Text. If the source artificact is, say, a "Microsoft Office File", this level of "inference" may just mean converting to some more convenient format. If the source artifact is a scanned page, then this interpetation might be done by OCR or a human transcriber. Similar for audio.
+
+We draw the concept of "tokens" from analytic philosophy. More on tokens at [the Stanford Encyclopedia of Philosophy](https://plato.stanford.edu/entries/types-tokens/#DisBetTypTok).
+
+At the level of *Tokens*, the processes that can produce inference include:
+
+- File format conversion
+- OCR
+- Human transcription
+
+###  3. Signifiers: *Deciding which Tokens are references to entities and attributes*
+
+On their own, Tokens encode claims about what language is present in a Text, but not about what sorts of things in the world that the Text might be describing. We must make inferences about Tokens to produce *Signifiers* – references to abstract types of entities and attributes contained in a Token.
```

Comment:
I was trying to express that Tokens are a type of inference made about words and characters (i.e. language), not about things in the world (i.e. entities). My goal was to make explicit that we're ascending a layer of abstraction. Does that make sense? 

"Claims about what language is present" is a clumsy way of putting it – more precise might be:

> Tokens encode claims about *the presence of information in a Text*, but do not describe what that information might "mean".